core/src/main/java/org/apache/calcite/rel/metadata/RelMetadataQuery.java

```diff
@@ -505,6 +509,29 @@ public static RelMetadataQuery instance() {
     }
   }
 
+  /**
+   * Returns the
+   * {@link BuiltInMetadata.ForeignKeys#getForeignKeys(boolean)}
+   * statistic.
+   *
+   * @param rel         the relational expression
+   * @param ignoreNulls if true, ignore null values when determining
+   *                    whether the keys are foreign keys
+   *
+   * @return            set of keys, or empty set if this information cannot be determined
+   *                    (whereas empty set indicates definitely no keys at all)
```

Comment:
In `BuiltInMetadata` you have a different javadoc for return, please check and align them

Reply:
Adjust them to be the same.